addons/models/auth/AuthPolicy.json

```diff
@@ -0,0 +1,394 @@
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": true,
+          "isOptional": false,
+          "isUnique": true,
+          "skipScrubbing": true,
+          "includeInNotification": true,
+          "indexTypeESFields": {
+            "text": {
+              "type": "text",
+              "analyzer": "atlan_text_analyzer"
+            }
+          }
+        },
+        {
+          "name": "name",
+          "typeName": "string",
+          "cardinality": "SINGLE",
+          "isIndexable": true,
+          "isOptional": false,
+          "isUnique": false,
+          "searchWeight": 10,
+          "skipScrubbing": true,
+          "indexTypeESConfig": {
+            "analyzer": "atlan_text_analyzer"
+          },
+          "indexTypeESFields": {
+            "keyword": {
+              "type": "keyword",
+              "normalizer": "atlan_normalizer"
+            },
+            "stemmed": {
+              "type": "text",
+              "analyzer": "atlan_text_stemmer"
+            }
+          }
+        },
+        {
+          "name": "description",
+          "typeName": "string",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": true,
+          "isUnique": false,
+          "searchWeight": 9,
+          "skipScrubbing": true,
+          "includeInNotification": true,
+          "indexTypeESConfig": {
+            "analyzer": "atlan_text_analyzer"
+          },
+          "indexTypeESFields": {
+            "keyword": {
+              "type": "keyword",
+              "normalizer": "atlan_normalizer"
+            }
+          }
+        },
+        {
+          "name": "tenantId",
+          "typeName": "string",
+          "cardinality": "SINGLE",
+          "isIndexable": true,
+          "isOptional": true,
+          "isUnique": false,
+          "skipScrubbing": true
+        },
+        {
+          "name": "policyType",
+          "typeName": "AuthPolicyType",
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": true,
+          "isUnique": false,
+          "skipScrubbing": true,
+          "includeInNotification": true
+        },
+        {
+          "name": "policyServiceName",
+          "typeName": "string",
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": true,
+          "isUnique": false,
+          "skipScrubbing": true,
+          "includeInNotification": true
+        },
+        {
+          "name": "policyCategory",
+          "typeName": "string",
+          "indexType": "STRING",
+          "cardinality": "SINGLE",
+          "isIndexable": false,
+          "isOptional": true,
+          "isUnique": false,
+          "skipScrubbing": true,
+          "includeInNotification": true
+        },
+        {
+          "name": "policySubCategory",
```

Comment:
ENUM

Reply:
Changing policySubCategory to ENUM will always require a value to create/update the entity as Atlas does not allow skipping or empty value for ENUM attribute
policySubCategory is only applicable in case of Persona/Purpose policies

Not changing this to ENUM